language/stdlib/modules/designated_dealer.move

```diff
@@ -0,0 +1,150 @@
+address 0x0 {
+    module DesignatedDealer {
+        use 0x0::LibraTimestamp;
+        use 0x0::Vector;
+        use 0x0::Transaction as Txn;
+        use 0x0::Libra;
+        use 0x0::LibraAccount;
+        use 0x0::Coin1;
+        use 0x0::Coin2;
+
+        resource struct Dealer {
```

Comment:
@tzakian Is there enough overlap with `AccountLimits` to use some the type/logic from there?